bug.yml

```diff
@@ -0,0 +1,61 @@
+---
+
+jira_server: "https://issues.stage.redhat.com"
```

Comment:
nice! the only suggestion i have is having jira and bugzilla as separate top level keys, then in future they can contain their own settings if we need them
```
jira:
   server: <>
   project:
bugzilla:
   server: <>
   classification: <>
   product: <>
```